x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -58,6 +59,11 @@ export function ServiceOverview({
         <SearchBar prepend={transactionTypeLabel} />
         <EuiPage>
           <EuiFlexGroup direction="column" gutterSize="s">
+            {isRumAgentName(agentName) && (
```

Comment:
Not a big deal, but we might as well call this once instead of a bunch of times.